.poshchan/settings.json

```diff
@@ -1,8 +1,18 @@
 {
-    "build_targets": {},
-    "authorized_users": [
-        "SteveL-MSFT",
-        "TravisEz13",
-        "anmenaga"
-    ]
+    "version": "0.1",
+    "azdevops": {
+        "organization": "powershell",
+        "project": "docker"
+    },
+    "build_targets": {
+        "CI": "Docker-PR"
+    },
+    "authorized_users": {
+        "build_targets": [
+            "SteveL-MSFT",
+            "TravisEz13",
+            "anmenaga"
+        ],
+        "reminders": "*"
```

Comment:
again, early preview.  Not everyone. @SteveL-MSFT 	how many people are we comfortable with?